polaris-migrator/src/migrations/admin-legacy-media-queries/transform.ts

```diff
@@ -0,0 +1,23 @@
+import type {API, FileInfo} from 'jscodeshift';
+import type {Plugin} from 'postcss';
+import postcss from 'postcss';
+
+export default async function transformer(file: FileInfo, _: API) {
+  return postcss(plugin()).process(file.source, {
+    syntax: require('postcss-scss'),
+  }).css;
+}
+
+const plugin = (): Plugin => {
+  return {
+    postcssPlugin: 'admin-legacy-media-queries',
+    AtRule(atRule) {
+      if (atRule.name === 'media' || atRule.name === 'container') {
+        atRule.params = atRule.params.replace(
+          /#\{media-queries\.\$(?<breakpoint>[^}]*)}/g,
```

Comment:
Can we make the namespace (e.g. `media-queries.`) a CLI flag? For example, see the [tokenize-font migration](https://github.com/Shopify/polaris/blob/046894b2c1c6696935f9e4b269c3fb53b6db9939/polaris-migrator/src/migrations/v9-styles-tokenize-font/transform.ts#L37) and associated [tests](https://github.com/Shopify/polaris/blob/046894b2c1c6696935f9e4b269c3fb53b6db9939/polaris-migrator/src/migrations/v9-styles-tokenize-font/tests/with-namespace.input.scss#L5).

Reply:
> do you have examples of some of those 1Ps I could check?

https://grokt.shopify.io/results?q=-repo%3AShopify%2Fpolaris+-repo%3AShopify%2Fweb+%5C%24p-breakpoints-

> I'd like to see if this can be applied to other codebases and how to generalize it

I think an optional namespace should be sufficient

> I think constructing the regex out of strings is going to get a bit more hairy than this

Let me know if you need assistance 👍